components/brave_wallet_ui/components/desktop/popup-modals/edit-visible-assets-modal/index.tsx

```diff
@@ -272,11 +272,17 @@ const EditVisibleAssetsModal = (props: Props) => {
   }
 
   const findUpdatedTokenInfo = (token: BraveWallet.BlockchainToken) => {
-    return updatedTokensList.find((t) => t.contractAddress.toLowerCase() === token.contractAddress.toLowerCase())
+    return updatedTokensList.find((t) =>
+      t.contractAddress.toLowerCase() === token.contractAddress.toLowerCase() &&
+      t.chainId === token.chainId)
   }
 
   const isUserToken = (token: BraveWallet.BlockchainToken) => {
-    return updatedTokensList.some(e => e.contractAddress.toLowerCase() === token.contractAddress.toLowerCase())
+    return updatedTokensList.some(e =>
```

Comment:
What's the reasoning for naming the iterated updatedToken variable `e`?

Reply:
Good point! not sure `e` was being used there, but changed to `t` for `token`.